src/yunohost/app.py

```diff
@@ -1933,8 +1933,14 @@ def _install_appslist_fetch_cron():
 
     logger.debug("Installing appslist fetch cron job")
 
+    cron_job = []
+    cron_job.append("#!/bin/bash")
+    # We add a random delay between 0 and 60 min to avoid every instance fetching
+    # the appslist at the same time every night
+    cron_job.append("(sleep $((RANDOM%3600)); yunohost app fetchlist > /dev/null 2>&1) &")
```

Comment:
please split this line in 2 separate line, one with the delay, and the other with the `fetchlist` command, to improve readability